docs/semgrep-code/semgrep-pro-engine-intro.md

```diff
@@ -138,11 +138,11 @@ To enable Semgrep Pro Engine in the Semgrep Cloud Platform, follow these steps:
 1. Sign in to [Semgrep Cloud Platform](https://semgrep.dev/login).
 1. Select **[Settings](https://semgrep.dev/orgs/-/settings)**.
 1. Enable the <i class="fa-solid fa-toggle-large-on"></i> **Semgrep Pro Engine beta** toggle.
-1. Ensure that you have the **default ruleset** added in your **[Rule Board](https://semgrep.dev/orgs/-/board)**. In the Rule Board, the **default ruleset** is in the **Monitor column**. If this ruleset is **not** added, go to [https://semgrep.dev/p/default](https://semgrep.dev/p/default), and then click **Add to Rule Board**.
+1. Ensure that you have the **default ruleset** added in your **[Rule Board](https://semgrep.dev/orgs/-/board)**. In the Rule Board, the **default ruleset** is recommended for the **Monitor column**. If this ruleset is **not** added, go to [https://semgrep.dev/p/default](https://semgrep.dev/p/default), and then click **Add to Rule Board**.
```

Comment:
For @armchairlinguist :

The difference between your change ("recommended for") and the existing content is that it's unclear if you're telling the user "put default ruleset in Monitor" or "default ruleset is typically found in Monitor". What the step was trying to say is that "this is where you usually find it, make sure it's there, if it's not there, add it and put it in Monitor." Hope this helps.

Reply:
I think the way it's currently written doesn't say what it's trying to either, though it's possible I didn't make things better. 😆 

"the default ruleset is in the Monitor column" to me means that it **is** there, and cannot/will not be anywhere else if you have added it.

The situation, as I understand it, is that default and comment are the two rulesets that are currently targeted for Pro rules. The recommended placement is Monitor for default and Comment for comment. If you don't have the default ruleset at all, you might not get many results. (This is the primary situation the direction is trying to avoid.) If you have it somewhere (but not in the recommended place), you will get results, but they might be noisy. So the most important thing is that you have it, but also the placement that is recommended is Monitor.

Perhaps one option is:

```suggestion
1. Ensure that you have the **default ruleset** added in your **[Rule Board](https://semgrep.dev/orgs/-/board)**. If this ruleset is **not** added, go to [https://semgrep.dev/p/default](https://semgrep.dev/p/default), and then click **Add to Rule Board**. For best results, this ruleset should be placed in the Monitor column.
```
Thoughts?

Reply:
Yes, agreed. Feel free to commit your suggestion.